packages/csharp/ArmoniK.Api.Common/Options/GrpcChannel.cs

```diff
@@ -40,4 +40,9 @@ public class GrpcChannel
   ///   Type of gRPC Socket used
   /// </summary>
   public GrpcSocketType SocketType { get; set; } = GrpcSocketType.UnixDomainSocket;
+
+  /// <summary>
+  ///   Port for communicate with this gRPC Channel in case a tcp socket is used
```

Comment:
`Address` should be used instead

Reply:
If I understood correctly, even if we choose to do the communication via a tcp socket, the agent should still create the unix socket because its absence is used to decide whether to stop other (listening) processes. This is why I added a separate field to hold the tcp port.

Reply:
Oh yeah there is that. To bypass this issue, I'd transform worker container in Kubernetes POD into a sidecar container (the same as for fluentbit) as it would guarantee that the worker starts before the agent and stops after it.

Reply:
As of today, the Kubernetes provider we use for terraform does not support to do so. However, with the direct use of Helm Charts, we will be able to do so.

Reply:
```suggestion
  ///   Port to communicate with this gRPC Channel in case a tcp socket is used
```

Reply:
@ngruelaneo thanks for the suggestion, this code was actually not necessary.